src/main/java/org/tron/core/services/http/FullNodeHttpApiService.java

```diff
@@ -0,0 +1,160 @@
+      context.addServlet(new ServletHolder(updateAccountServlet), "/updateaccount");
+      context.addServlet(new ServletHolder(voteWitnessAccountServlet), "/votewitnessaccount");
+      context.addServlet(new ServletHolder(createAssetIssueServlet), "/createassetissue");
+      context.addServlet(new ServletHolder(updateWitnessServlet), "/updatewitness");
+      context.addServlet(new ServletHolder(createAccountServlet), "/createaccount");
+      context.addServlet(new ServletHolder(createWitnessServlet), "/createwitness");
+      context.addServlet(new ServletHolder(transferAssetServlet), "/transferasset");
+      context.addServlet(new ServletHolder(participateAssetIssueServlet), "/participateassetissue");
+      context.addServlet(new ServletHolder(freezeBalanceServlet), "/freezebalance");
+      context.addServlet(new ServletHolder(unFreezeBalanceServlet), "/unfreezebalance");
+      context.addServlet(new ServletHolder(unFreezeAssetServlet), "/unfreezeasset");
+      context.addServlet(new ServletHolder(withdrawBalanceServlet), "/withdrawbalance");
+      context.addServlet(new ServletHolder(updateAssetServlet), "/updateasset");
+      context.addServlet(new ServletHolder(listNodesServlet), "/listnodes");
+      context
+          .addServlet(new ServletHolder(getAssetIssueByAccountServlet), "/getassetissuebyaccount");
+      context.addServlet(new ServletHolder(getAccountNetServlet), "/getaccountnet");
+      context.addServlet(new ServletHolder(getAssetIssueByNameServlet), "/getassetissuebyname");
+      context.addServlet(new ServletHolder(getNowBlockServlet), "/getnowblock");
+      context.addServlet(new ServletHolder(getBlockByNumServlet), "/getblockbynum");
+      context.addServlet(new ServletHolder(getBlockByIdServlet), "/getblockbyid");
+      context.addServlet(new ServletHolder(getBlockByLimitNextServlet), "/getblockbylimitnext");
+      context.addServlet(new ServletHolder(getBlockByLatestNumServlet), "/getblockbylatestnum");
+      context.addServlet(new ServletHolder(getTransactionByIdServlet), "/gettransactionbyid");
+      context.addServlet(new ServletHolder(listWitnessesServlet), "/listwitnesses");
+      context.addServlet(new ServletHolder(getAssetIssueListServlet), "/getassetissuelist");
+      context.addServlet(new ServletHolder(getPaginatedAssetIssueListServlet),
+          "/getpaginatedassetissuelist");
+      context.addServlet(new ServletHolder(totalTransactionServlet), "/totaltransaction");
+      context
+          .addServlet(new ServletHolder(getNextMaintenanceTimeServlet), "/getnextmaintenancetime");
+      context.addServlet(new ServletHolder(createAddressServlet), "/createaddress");
+      context.addServlet(new ServletHolder(easyTransferServlet), "/easytransfer");
+      context.addServlet(new ServletHolder(generateAddressServlet), "/generateaddress");
+      server.start();
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Use a logger to log this exception. ([squid:S1148](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1148))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1114&issues=AWRjYLgrMt9UOhrw8p6v&open=AWRjYLgrMt9UOhrw8p6v)

[ISSUE_KEY]: AWRjYLgrMt9UOhrw8p6v